scripts/specializations.json

```diff
@@ -1,11 +1,13 @@
 {
 	"specializations": [
 		{
+			"full_batch_size": "4",
```

Comment:
does this mean we only test with full_batch_size? I think we should test both scenarios separately

Reply:
This will work with non-CB also. It's not an error to have extra symbols listed in the specializations.json.